app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/AuthAction.scala

```diff
@@ -47,6 +47,7 @@ class AuthActionImpl @Inject() (
 ) extends AuthActionBase(authConnector, allowedUsers, metrics, mcc, appConfig) with AuthAction {
   override val mustBeEnrolled: Boolean                 = false
   override val redirectEnrolledUsersToReturns: Boolean = true
+  override val agentsAllowed: Boolean                  = false
 }
 
 class AuthNoEnrolmentCheckActionImpl @Inject() (
```

Comment:
It would be good (perhaps when there's less churn in the codebase) to change these names to better reflect their new responsibilities.